models/models_methods.go

```diff
@@ -828,6 +819,48 @@ func validateDAGPipeline(pipelineID string, pipeline *PipelineModel, config *Bit
 	return validateGraph(pipeline)
 }
 
+func validateParallelizedWorkflow(pipelineID, workflowID string, workflow GraphPipelineWorkflowModel, config *BitriseDataModel) error {
+	isDynamic := false
+	parallel, err := strconv.Atoi(workflow.Parallel)
+	if err != nil {
+		re := regexp.MustCompile(`^\$[a-zA-Z_][a-zA-Z0-9_]*$`)
```

Comment:
Do we need to have this regexp? I only check that the string starts with a `$` sign in the pipeline service. If it starts with that then we can be sure it is an env var and the regular env var naming conventions apply. Which we might not need to validate.

Reply:
Not sure I get this, do you mean that we would get validation "for free" at some level already? Or that we don't care about that? What happens if the user puts `$my_env_var_with_a-typo` here?